generated/Category.h

```diff
@@ -0,0 +1,43 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_Category_H_
+#define ORG_OPENAPITOOLS_MODEL_Category_H_
+
+#include "serialization.h"
+
+#include <optional>
+
+#include <string>
+
+namespace org {
+namespace openapitools {
+namespace model {
+
+/**
+ * \brief A category for a pet
+ */
+struct Category {
+    static const char *dtoName() { return "swagger::dto::Category"; }
+
+    // required fields
+
+    // optional fields
```

Comment:
In the example yaml file, none of these are `required`. Not sure about our real DTOs, but are we OK with such behavior, in general? On the other hand, if it isn't marked `required`, there's not much we can do :sweat_smile:

Reply:
What kind of behavior do you mean?

Reply:
I mean a DTO having all fields optional. But thinking about it now, there's really nothing we can do, if no fields are marked required.

Reply:
Yeah, we constantly tell the backend off for not marking fields as "required". But it sort of started becoming second-nature for them by now. So yeah: if its not "required", unfortunately that implies that its "optional".